### PR TITLE
Fix docs and tests

### PR DESCRIPTION
**Pull request by pedrorijo91**
- Fixed badges
- Fixed contributors
- Added usage example
- Added tests

### CODE REVIEW

File: src/test/scala/com/codacy/api/FileHelperTest.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        projectToken shouldNot be('empty)
+        commitUUID shouldNot be('empty)
+
+        Right(true)
+    }
+
+    result should be('right)
+
+    result match {
```
Comment (mrfyda) [inline src/test/scala/com/codacy/api/FileHelperTest.scala]: Codacy Issue found: [Default case not specified.](https://www.codacy.com/app/Codacy/codacy-api-scala/file/1246697786/issues/source?bid=2272948&fileBranchId=2935685#l21)

Comment (mrfyda) [inline src/test/scala/com/codacy/api/FileHelperTest.scala]: Codacy Issue found: [Default case not specified.](https://www.codacy.com/app/Codacy/codacy-api-scala/file/1247389679/issues/source?bid=2272948&fileBranchId=2935685#l21)

**Comment by machadoit** [comment]
delete travis.yml
add java version to circle.yml

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
-[![Build Status](https://travis-ci.org/codacy/codacy-api-scala.svg?branch=master)](https://travis-ci.org/codacy/codacy-api-scala)
+[![Circle CI](https://circleci.com/gh/codacy/codacy-api-scala/tree/master.svg?style=shield)](https://circleci.com/gh/codacy/codacy-api-scala/tree/master)
 [![Codacy Badge](https://www.codacy.com/project/badge/650fe924dba349458ee29d44f07dae6c)](https://www.codacy.com/app/Codacy/codacy-api-scala)
```
Comment (machadoit) [inline README.md:2]: add /grade

File: src/test/scala/com/codacy/api/FileHelperTest.scala [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      case (projectToken, commitUUID) =>
+
+        projectToken shouldNot be('empty)
+        commitUUID shouldNot be('empty)
+
+        Right(true)
+    }
+
+    result should be('right)
+
```
Comment (machadoit) [inline src/test/scala/com/codacy/api/FileHelperTest.scala:20]: I think we should test this cases with (import org.scalatest.EitherValues._):

either.right.value should be (true)

File: src/test/scala/com/codacy/api/FileHelperTest.scala (hunk 1 above)
Comment (mrfyda) [inline src/test/scala/com/codacy/api/FileHelperTest.scala:21]: Codacy Issue found: [Default case not specified.](https://www.codacy.com/app/Codacy/codacy-api-scala/file/1292018471/issues/source?bid=2272948&fileBranchId=2935685#l21)

**Pull request merged**